yt/data_objects/construction_data_containers.py

```diff
@@ -254,9 +263,12 @@ def get_data(self, fields=None):
         projected_units = self.comm.mpi_bcast(self._projected_units)
         self._projected_units = projected_units
         # Note that this will briefly double RAM usage
-        if self.method == "mip":
+        if self.method in ["max","mip"]:
             merge_style = -1
             op = "max"
+        if self.method == "min":
```

Comment:
This is the line that's breaking your docs recipe:
```suggestion
        elif self.method == "min":
```

Reply:
btw it'd be helpful to add an actual error message in the `NotImplementedError` raised a couple lines below this, like
```python
raise NotImlementedError(f"Unknown projection method {self.method!r}")
```

Reply:
Sooo I did a `grep -rin "NotImplementedError" .` from the root level of my yt-fork directory and `yt/visualization/volume_rendering/old_camera.py doesn't print anything, but it does log...do you think I should add a log message of the error message here too? Or an error message in the NotImplementedError over there?

Still going through the other instances :)

Reply:
Ran out of steam but will return to this later.

Reply:
Ok maybe I shouldn't have made this suggestion, but please try and keep this PR's scope tight. Another PR adding actual error messages to bare `raise NotImplementedError` calls would be welcome but it's hardly related to what you're doing here and we don't want to mix up everything, as the bigger the PR, the harder it is to review

Reply:
Sure, that makes sense. I'll try to hold you to that then :P